modules/ROOT/pages/java-support.adoc

```diff
@@ -0,0 +1,160 @@
+
+Follow these best practices:
+
+* Upgrade your standalone Mule instances.
+* Upgrade on-premises Mule instances that are managed through Runtime Manager.
+* Update the Mule version used to build your Mule applications.
+* xref:mule-runtime::deploying.adoc[Deploy your applications] to a target running on Mule 4.6 or later.
+
+[[mule-apps]]
+== Mule Apps
+
+The majority of Mule applications currently in use can be updated, tested, and redeployed. However, any apps and connectors that are running custom Java code require additional work to certify those components. Before you upgrade your integration apps or Mule Gateway policies and proxies to Java 17, all extensions, modules, and connectors used within those apps and policies must first be updated to be Java 17-compatible. 
+
+[IMPORTANT]
+To ensure your API proxies or Mule apps are protected when upgrading, upgrade your API policies *before* upgrading your API proxies or Mule apps.
+
+[[application-performance]]
+=== Application Performance
+
+Java 17 manages memory differently than it did in earlier versions like Java 8. In some cases, Java 17 preferentially commits to heap memory or uses heap memory for structures in some apps, which earlier Java versions allocated to native memory.
+
+Because of this change in Java 17 memory management, it is important for you to conduct xref:mule-runtime::tuning-test-validations.adoc[performance testing] and xref:mule-runtime::tuning-monitoring.adoc[monitoring] for your most common workloads.
+
+If your apps are deployed on-premises: 
+
+* If your workloads are running on Runtime Fabric, you must have 1400 MB of memory allocation for 0.50 vCPU replica sizes. Memory requirements vary for individual apps. 
+* If your workloads are running on-premises in a hybrid or standalone environment, you must tune your operating system's garbage collection algorithm. 
+
+
+[[anypoint-connectors-and-modules]]
+== Anypoint Connectors and Modules
+
+Anypoint Connectors and modules are certified by MuleSoft to run on Java 17. For information about Java 17 support for each connector and module, see the xref:release-notes::connector/anypoint-connector-release-notes.adoc[Release Notes] for that Mule 4 connector or module. Java compatibility information is also provided on the Exchange tiles for connectors and modules. 
+
+The most commonly used Anypoint Connectors and modules are Java 17-compatible now, and others are being released regularly. For a list of Java 17-compatible connectors, see https://help.salesforce.com/s/articleView?id=000782248&type=1[Java 17 Compatible Anypoint Connectors^].
+
+To upgrade Anypoint Connectors:
+
+. Update your Mule app to run on Mule runtime 4.6 or later. 
+. Change the target JVM at the project property level. This includes both upgrades and downgrades of the JVM version.
```

Comment:
"This" needs a noun.
https://confluence.internal.salesforce.com/display/MTDT/MuleSoft+CX+Writing+Style+Reference#MuleSoftCXWritingStyleReference-%22it,%22%22this,%22%22these%22